tests/MongoDB.Driver.Tests/MongoDatabaseImplTests.cs

```diff
@@ -34,14 +35,16 @@ namespace MongoDB.Driver
 {
     public class MongoDatabaseImplTests
     {
+        private IMongoClient _client;
```

Comment:
I hate adding a new field like this when I consider them to be bad, but it was going to result in more changes to this file than I wanted to do to remove all these fields.